### ISSUE TITLE
Scope Identify Onboarding Processes/Tools

### ISSUE THREAD

**Issue by MichaelKohler** [mozilla/Reps#120]
## Goal: 
The Reps Onboarding process we are currently experimenting with is clearly documented in terms of process and tools and the "old" process is summarized. And Michael is in sync with Christos about scope and timeline of this project.

**This supports the Community Development Team:**
Objective 3: We start to have a unified approach across Mozilla of recruiting, aligning and investing in mobilizers (including the Reps program) that is showing direct value for business objectives
* KR1: Shared onboarding framework and tools are identified for Reps, Regional Coaches, Club Leads and Resource Leads [shared with MoFo]

## Roles:
Responsible: Michael Kohler
Accountable: Reps Council

## Required: 
- [x] Sync with Christos on scope and timeline (@MichaelKohler, Due: Jan 25th) 
- [x] Document onboarding process and tools (Reps) (@MichaelKohler, Due: Jan 31st)

**Comment by MichaelKohler** [comment]
Processes and Tools have been documented here: https://wiki.mozilla.org/ReMo/SOPs/Joining

I will create another issue here if necessary. For now I'm blocked on Christos to provide the rest of the data. Then I can help him analyze them.
